code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -63,18 +63,16 @@ \subsection{Table of Symbols}
 \\
 $A$ & Area: A part of an object or surface & $\text{m}^{2}$
 \\
-$b$ & Base Width of a Slice: in the x-direction & m
+$\mathbf{b}$ & Base Width of a Slice: in the x-direction & m
```

Comment:
The description is given as "Base width of a slice", but that implies the base width of one slice.  I think it should be base width of slices, to show that there is a sequence of slices.  

At some point we really are going to have to tackle how best to handle sequences, because sometimes we will want to reference a specific slice's base width, and other times we mean all of the slices, or maybe some sub-sequence of slices.